cli/src/pcluster/models/cluster.py

```diff
@@ -287,7 +287,7 @@ def login_nodes_status(self):
         """Status of the login nodes pool."""
         login_nodes_status = LoginNodesStatus(self.stack_name)
         if self.stack.scheduler == "slurm" and self.config.login_nodes:
-            login_nodes_status.retrieve_data()
+            login_nodes_status.retrieve_data(self.config.login_nodes.pools[0].name)
```

Comment:
Add a comment saying that this works because by design we have now only one pool.
But we should fix this.